src/comfyui-refiners/README.md

```diff
@@ -25,7 +25,13 @@ To easily install the nodes, run the following command:
 comfy node registry-install comfyui-refiners
 ```
 
-You may also download the nodes by cliking the "Download Latest" button and unzipping the content of the archive into you custom_nodes directory.
+To manually install the nodes, you may alternatively do the following:
```

Comment:
I would create two sub-sections (like `### Comfy Registry (recommended)` and `### Manual`).